src/main/kotlin/no/nav/syfo/melding/api/MeldingApi.kt

```diff
@@ -34,6 +36,22 @@ fun Route.registerMeldingApi(
             )
         }
 
+        get("/{$msgId}/{$vedleggNumber}/pdf") {
```

Comment:
Er det noen spesiell grunn til å bruke msgId som parameter her? Tenker det ville vært mer naturlig om man apiet kunne hente vedlegg for en Melding med gitt UUID.